### ISSUE TITLE
Rule option suggestion: Quotemark backtick

### ISSUE THREAD

**Issue by ShayBox** [palantir/tslint#3814]
### Bug Report

- __TSLint version__: 5.9.1
- __TypeScript version__: 2.8.1
- __Running TSLint via__: VSCode

#### TypeScript code being linted

```ts
console.log(`Hello World`);
```

with `tslint.json` configuration:

```json
{
	"rules": {
		"quotemark": [true, "backtick"]
	}
}
```

#### Actual behavior

\' should be " (quotemark)

#### Expected behavior

Should allow you to use backticks

This is a rule option suggestion, to allow `backtick` as an option like single and double, like eslint allows.

**Comment by ericferreiralife** [comment]
This seems like a duplicate of #539

**Comment by johnwiseheart** [comment]
Agreed this is a duplicate of #539.
